pandas/tests/groupby/test_categorical.py

```diff
@@ -67,6 +68,12 @@ def setUp(self):
              'E': np.random.randn(11),
              'F': np.random.randn(11)})
 
+    def test_level_groupby_get_group(self):
+        # gh15155
+        testGroupBy = self.mframe.groupby(level=["first"])
+        assert_numpy_array_equal(testGroupBy.get_group("foo").values,
```

Comment:
this is a separate issue. please post an issue with a simpe repro.

for now you can simply construct the expected frame directly for comparision, and put a note with a reference to the new issue.